pillar.sls.example

```diff
@@ -0,0 +1,48 @@
+{#
+SPDX-FileCopyrightText: 2024 seven-beep <ebn@entreparentheses.xyz>
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+#}
+{#
+
+Example of a pillar state structure for https://github.com/ben-grande/dotfiles
+
+This file need to be placed in your pillar_roots, renamed with a '.sls'
+extension and referenced by a top file. See 'pillar.top.example' for an example
+and the README for instructions.
+
+Each entry in the pillar state is optional. You can disable the entire dotfile
+setup by keeping only 'qusal:dotfiles:all' to 'false', or selectively disable
+individual components by setting 'false' for the corresponding state entries.
+
+#}
+
+qusal:
+  dotfiles:
+    # Control whether or not applying the copy-all.sls or any other dotfiles states.
+    # Disabling it will disable all states, unless some are specifically whitelisted.
+    all: true
+    # Control whether or not applying the copy-x11.sls state.
```

Comment:
This is wrong, x11 is not near this line.
Do you feel it is needed to repeat the same comments across all lines?

I think it is better if we generalize with a paragraph above, after line 16.

Reply:
No I neither think this was necessary.

Adjustments were made.